.gitignore

```diff
@@ -1,5 +1,6 @@
 build*/
 _build/
+!build_apps/
```

Comment:
I don't think this is necessary - it wasn't previously ignored?

Reply:
It is necessary as any file created under `build_apps/` will not be visible to git, because of the `build*/` exclusion rule two lines above (note that it doesn't affect existing files). I'm modifying this to `doc/build_apps/` to be more specific though.